### PR TITLE
Vuln Version !

### PR DESCRIPTION
**Pull request by LaCapitainerie**
In this update i **added Vuln Version number**, **fix some bug**, and tested the update for all possible exports
In order to continue working on it i take the liberty of **typing all classes** because it was pretty annoying to look at the definition and forgot where i was..

### Excel By Vuln
image

### Excel By Host
image

### Word By Vuln
image

### CODE REVIEW

File: openvasreporting/libs/parsed_data.py [hunk 1]
```diff
@@ -489,13 +504,13 @@ def __init__(self, vuln_id, name, threat, **kwargs):
...
-        references = kwargs.get("references", "Uknown") or "Unknown"
-        family = kwargs.get("family", "Unknown") or "Unknown"
-        result = kwargs.get("description", "Unknown") or "Unknown"
+        cves: list[str] = kwargs.get("cves", list()) or list()
+        cvss:     float = kwargs.get("cvss", -1.0) or -1.0
+        level:      str = kwargs.get("level", "None") or "None"
+        tags:      dict = kwargs.get("tags", dict()) or dict()
+        references: str = kwargs.get("references", "Unknown") or "Unknown"
+        family:     str = kwargs.get("family", "Unknown") or "Unknown"
+        result:     str = kwargs.get("description", "Unknown") or "Unknown"
```
Comment (TheGroundZero) [inline openvasreporting/libs/parsed_data.py:513]: Adding spaces to get `=` aligned might conflict with PEP 8
https://peps.python.org/pep-0008/#whitespace-in-expressions-and-statements

File: openvasreporting/libs/parsed_data.py [hunk 2]
```diff
@@ -573,22 +585,23 @@ def __eq__(self, other):
...
+                other.solution      != self.solution      or
                 other.solution_type != self.solution_type or
-                other.references != self.references or
-                other.threat != self.threat or
-                other.family != self.family or
-                other.result != self.result
+                other.references    != self.references    or
+                other.threat        != self.threat        or
+                other.family        != self.family        or
+                other.result        != self.result
```
Comment (TheGroundZero) [inline openvasreporting/libs/parsed_data.py:604]: Same comment re. PEP 8
https://peps.python.org/pep-0008/#whitespace-in-expressions-and-statements

**Comment by LaCapitainerie** [comment]
Yes i forgot about that, fixed it 👍

**Comment by TheGroundZero** [comment]
Looks good. Thank you for your contribution!
I don't use OpenVAS myself anymore so can't really compare. So if problems arise because of this change, I may need to forward people to you instead ;-)

Can you up the version to `1.6.0` and update the Changelog please?
https://github.com/TheGroundZero/openvasreporting/blob/master/setup.cfg / https://github.com/TheGroundZero/openvasreporting/blob/master/docs/conf.py
https://github.com/TheGroundZero/openvasreporting/blob/master/CHANGELOG / https://github.com/TheGroundZero/openvasreporting/blob/master/docs/changelog.rst

**Comment by LaCapitainerie** [comment]
> Looks good. Thank you for your contribution! I don't use OpenVAS myself anymore so can't really compare. So if problems arise because of this change, I may need to forward people to you instead ;-)

Sure ! I'm currently working and trying to resolve all the issues that are still open, could you please close or add a message on those where you were speaking with the author of the issue ? (To know if the problem is still on)

**Pull request merged**